packages/react-native/Libraries/StyleSheet/StyleSheetTypes.d.ts

```diff
@@ -271,6 +271,7 @@ export interface ViewStyle extends FlexStyle, ShadowStyleIOS, TransformsStyle {
    * Controls whether the View can be the target of touch events.
    */
   pointerEvents?: 'box-none' | 'none' | 'box-only' | 'auto' | undefined;
+  transformOrigin?: Array<string | number> | string | undefined;
```

Comment:
@jacobp100 I think we should only support [spec](https://www.w3.org/TR/css-transforms-1/) compliant value here so string should be good!

Reply:
I think we have to accept a JS form as well so stuff like `Animated` works. Just working on getting the string version to pass the spec!